test/example/c-api-kitchen-sink.c

```diff
@@ -263,8 +263,7 @@ void test_features() {
   printf("BinaryenFeatureMemory64: %d\n", BinaryenFeatureMemory64());
   printf("BinaryenFeatureTypedFunctionReferences: %d\n",
          BinaryenFeatureTypedFunctionReferences());
-  printf("BinaryenFeatureRelaxedSIMD: %d\n",
-         BinaryenFeatureRelaxedSIMD());
+  printf("BinaryenFeatureRelaxedSIMD: %d\n", BinaryenFeatureRelaxedSIMD());
```

Comment:
No, this is just that a SIMD PR landed without running clang-format on the test suite. (We don't check that on CI, it seems?)

Reply:
I guess the issue is that new files keep showing up when I run
```
git clang-format origin/main
```
locally. I assume for everyone else too? It seems best to keep landing formatting fixes for them to avoid that. (Finding a way to ignore them would also work, but there seems little harm in these incremental updates, that at least the code does not get worse?)

Reply:
What new files are you seeing? I would generally be in favor of formatting test/ and I think a big formatting PR + CI coverage would be a good path forward that minimizes the amount we have to think about it.

Reply:
> git clang-format origin/main

I'm not sure why this formats the test file, because this only formats the diff... But yeah, I'm also in favor of formatting test/.

Reply:
> What new files are you seeing? 

Sorry, maybe I was unclear. When someone lands a new change to `tests/` that contains unformatted code, then that code ends up showing up in my next diff.

While we decide about formatting all of `test/` or not, I think it would be good to keep landing formatting changes to new changes there, as otherwise (1) I will need to manually ignore some of the results of clang-format locally, and (2) it seems silly to ignore them to keep them unformatted/less good...

Reply:
(I am also in favor of formatting all of `test/` btw)